decksite/data/archetype.py

```diff
@@ -87,7 +90,8 @@ def load_archetypes_deckless(order_by: str = '`num_decks` DESC, `wins` DESC, nam
             aca.ancestor -- aca.ancestor will be unique per a.id because of integrity constraints enforced elsewhere (each archetype has one ancestor) but we let the database know here.
         ORDER BY
             {order_by}
-    """.format(season_query=query.season_query(season_id), order_by=order_by)
+    """.format(tournament='_nonleague' if tournament_only else '',
```

Comment:
Although it's irrelevant given that all our match data is from league or tournaments right now maybe call this `_tournaments` not `_nonleague`?

Reply:
Will fix.